doc/manuals/devel/cprs.md

```diff
@@ -34,14 +35,16 @@ Note that there are a number of service routines that end up calling `postUpdate
 
 * All attributes in the incoming payload are marked as **Not Found** (step 1).
 * [**mongoBackend** library](README.md#srclibmongobackend) processes the request (see diagrams [MB-01](mongoBackend.md#flow-mb-01) or [MB-02](mongoBackend.md#flow-mb-02)) and marks all attributes in the requests in one of three possible ways (step 2):
-       * Not Found
-       * Found in Local Broker
-       * Found in Remote Context Provider
-* The attributes that are found in a remote context provider need to be forwarded. The local attributes are simply updates while those not found are marked as such in the response.
-* A new vector of `ContextElementResponse` is created and filled with all those attributes that are to be forwarded (step 3). These responses are then added to the response vector that was output from **mongoBackend**. If no attribute is "found", then the `ContextElementResponse` is prepared with a 404 Not Found.
+
+    * Not Found
```

Comment:
Have you tested this render correctly in github web? List within list are tricky in markdown...